src/components/Link.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import classNames from "classnames";
+import PropTypes from "prop-types";
+
+import { ComponentRegister, LinkComponent } from "../ComponentRegister";
+
+import "lbh-frontend/lbh/core/_links.scss";
+
+/**
+ * A component for creating an anchor linking to other pages. You can customize the component used internally for
+ * integration with routers using {@link ComponentRegister.init}. By default, it uses an <a> tag.
+ *
+ * @param {string} [className]
+ * @param {string} href
+ * @param {ReactNode} children
+ *
+ * @returns {LinkComponent}
+ */
+export const Link: LinkComponent = ({ className, href, children }) => {
```

Comment:
I know in practice it does implement `LinkComponent` exactly, it would be expected for it to change in the future. For instance, we might want to start passing in an `onClick` handler that does AJAX stuff. So this should have its own interface. Probably `FunctionComponent<LinkProps>` where `LinkProps` defines the expected proptypes for this component, not the component it wraps.